packages/storybook/src/nimble/combobox/combobox-opened-matrix.stories.ts

```diff
@@ -45,28 +58,54 @@ if (remaining.length > 0) {
     throw new Error('New backgrounds need to be supported');
 }
 
+// prettier-ignore
```

Comment:
Are all these prettier-ignores really needed? How poorly is it formatting a couple lines?

Reply:
Apparently they made no difference. Removed.